src/axom/slam/policies/IndirectionPolicies.hpp

```diff
@@ -272,6 +277,66 @@ struct CoreArrayIndirection
   IndirectionBufferType* m_vecBuf;
 };
 
+/**
+ * \brief A policy class for sets with axom::ArrayView-based indirection
+ */
+template <typename PositionType, typename ElementType>
+struct ViewIndirection
```

Comment:
Minor: Should this class be `ArrayViewIndirection`  instead of `ViewIndirection`?